Classes/Service/ExtensionService.php

```diff
@@ -94,4 +95,16 @@ public function isStoragePathConfigured(): bool
     {
         return !Environment::isComposerMode() || count($this->resolveStoragePaths()) > 0;
     }
+
+    /**
+     * Returns, if typo3/cms-composer-installers v4 is used by checking, if ext:extension_builder is
```

Comment:
- Returns what?
- `ext:extension_builder` should be written simply like "extension"

Reply:
No need to return `what`, since its a function with a boolean return type.